cli/src/main/java/edu/stanford/futuredata/macrobase/pipeline/CubePipeline.java

```diff
@@ -0,0 +1,122 @@
+package edu.stanford.futuredata.macrobase.pipeline;
+
+import edu.stanford.futuredata.macrobase.analysis.classify.ArithmeticClassifier;
+import edu.stanford.futuredata.macrobase.analysis.classify.CubeClassifier;
+import edu.stanford.futuredata.macrobase.analysis.summary.APrioriSummarizer;
+import edu.stanford.futuredata.macrobase.analysis.summary.BatchSummarizer;
+import edu.stanford.futuredata.macrobase.analysis.summary.Explanation;
+import edu.stanford.futuredata.macrobase.analysis.summary.FPGrowthSummarizer;
+import edu.stanford.futuredata.macrobase.conf.Config;
+import edu.stanford.futuredata.macrobase.datamodel.DataFrame;
+import edu.stanford.futuredata.macrobase.datamodel.Schema;
+import edu.stanford.futuredata.macrobase.ingest.CSVDataFrameLoader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Default pipeline for cubed data: load, classify, and then explain
+ */
+public class CubePipeline implements Pipeline {
+    Logger log = LoggerFactory.getLogger("CubePipeline");
+    private String inputFile;
+
+    private String classifierType;
+
+    private double percentile;
+    private boolean includeHi;
+    private boolean includeLo;
+    private String countColumn;
+    private String meanColumn;
+    private String stdColumn;
+
+    private List<String> attributes;
+    private double minSupport;
+    private double minRiskRatio;
+
+
+    public CubePipeline(Config conf) {
+        inputFile = conf.getAs("inputFile");
+
+        classifierType = conf.getAs("classifier");
+
+        percentile = conf.getAs("percentile");
+        includeHi = conf.getAs("includeHi");
+        includeLo = conf.getAs("includeLo");
+        countColumn = conf.getAs("countColumn");
+        meanColumn = conf.getAs("meanColumn");
+        stdColumn = conf.getAs("stdColumn");
+
+        attributes = conf.getAs("attributes");
+        minSupport = conf.getAs("minSupport");
+        minRiskRatio = conf.getAs("minRiskRatio");
+    }
+
+    @Override
+    public void run() throws Exception {
+        Map<String, Schema.ColType> colTypes = getColTypes();
+        CSVDataFrameLoader loader = new CSVDataFrameLoader(inputFile);
+        loader.setColumnTypes(colTypes);
+        long startTime = System.currentTimeMillis();
+        DataFrame df = loader.load();
+        long elapsed = System.currentTimeMillis() - startTime;
+        log.info("Loading time: {}", elapsed);
```

Comment:
Use debug for diagnostic data, info for things normal end users might actually care about.